backend/src/main/kotlin/me/andrewda/controllers/RequestController.kt

```diff
@@ -21,8 +21,16 @@ object RequestController {
                 quantity = request.quantity
             }
 
-            if (request.fulfilled != null) {
-                fulfilled = request.fulfilled
+            if (request.funds != null) {
+                funds = request.funds
+            }
+
+            if (request.ordered != null) {
+                ordered = request.ordered
+            }
+
+            if (request.delivered != null) {
+                delivered = request.delivered
```

Comment:
This can all be removed. `funds`, `ordered` and `delivered` will never be made in the initial creation of a request.